frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/useShowDependentFeatures.ts

```diff
@@ -5,5 +5,5 @@ export const useShowDependentFeatures = (project: string) => {
     const { dependenciesExist } = useCheckDependenciesExist(project);
     const { isOss } = useUiConfig();
 
-    return Boolean(isOss() ? dependenciesExist : true);
+    return Boolean(project) && Boolean(isOss() ? dependenciesExist : true);
```

Comment:
dependent features were checking pending change requests under the hood but they were checking them for the empty project. The empty project came from the placeholder used in useFeature. After this fix if the useFeature returns the placeholder project (with empty project) we don't show dependencies section.